pctasks/core/tests/models/test_storage_event.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+from pctasks.core.cosmos.containers.storage_events import StorageEventsContainer
+from pctasks.core.models.event import StorageEvent, StorageEventRecord, StorageEventType
+
+
+@pytest.fixture
+def event_body():
+    body = {
+        "source": "/subscriptions/{subscription-id}/resourceGroups/Storage/providers/Microsoft.Storage/storageAccounts/my-storage-account",  # noqa: E501
+        "subject": "/blobServices/default/containers/my-file-system/blobs/new-file.txt",
+        "type": "Microsoft.Storage.BlobCreated",
+        "time": "2017-06-26T18:41:00.9584103Z",
+        "id": "831e1650-001e-001b-66ab-eeb76e069631",
+        "data": {
+            "api": "CreateFile",
+            "clientRequestId": "6d79dbfb-0e37-4fc4-981f-442c9ca65760",
+            "requestId": "831e1650-001e-001b-66ab-eeb76e000000",
+            "eTag": '"0x8D4BCC2E4835CD0"',
+            "contentType": "text/plain",
+            "contentLength": 0,
+            "contentOffset": 0,
+            "blobType": "BlockBlob",
+            "url": "https://my-storage-account.dfs.core.windows.net/my-file-system/new-file.txt",  # noqa: E501
+            "sequencer": "00000000000004420000000000028963",
+            "storageDiagnostics": {"batchId": "b68529f3-68cd-4744-baa4-3c0498ec19f0"},
+        },
+        "specversion": "1.0",
+    }
+    return body
+
+
+def test_storage_event_created(event_body):
+    event = StorageEvent.parse_obj(event_body)
+    assert event.time == "2017-06-26T18:41:00.9584103Z"
+    assert event.type == StorageEventType.CREATED
+    assert event.data.api == "CreateFile"
+    assert (
+        event.data.url
+        == "https://my-storage-account.dfs.core.windows.net/my-file-system/new-file.txt"
+    )  # noqa: E501
+
+
+def test_storage_events_record(event_body):
+    record = StorageEventRecord.parse_obj(event_body)
+
+    assert record.get_id() == "831e1650-001e-001b-66ab-eeb76e069631"
+    assert StorageEventRecord.migrate(event_body)
+
+
+@pytest.mark.cosmos
+class TestStorageEventsContainer:
+    def test_put_get_item(self, event_body):
```

Comment:
This test is [failing](https://github.com/microsoft/planetary-computer-tasks/actions/runs/4641272031/jobs/8214067850?pr=169) with

```pytb
/usr/local/lib/python3.10/dist-packages/azure/cosmos/_synchronized_request.py:148: CosmosResourceNotFoundError
```

The URL it's using is `parse_result = ParseResult(scheme='https', netloc='cdb-pctasks-ci-eastus.documents.azure.com:443', path='/dbs/pctasks/colls/tmp-storage-events-4641272031/', params='', query='', fragment='')`. Note the `tmp` and `4641272031`, which indicates that `PCTASKS_COSMOSDB__TEST_CONTAINER_SUFFIX` is set.

I'm not sure when that cosmos container is created.